### ISSUE TITLE
[Bug]: Could not find 'Xonotic'! (error 11)

### ISSUE THREAD

**Issue by SonaliBendre** [Calinou/scoop-games#980]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

xonotic

### Expected/Current Behaviour

```
Could not find 'Xonotic'! (error 11)
At D:\\scoop\apps\scoop\current\lib\core.ps1:638 char:9
+         throw "Could not find '$(fname $from)'! (error $($proc.ExitCo ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Could not find 'Xonotic'! (error 11):String) [], RuntimeException
    + FullyQualifiedErrorId : Could not find 'Xonotic'! (error 11)
```

### Steps to Reproduce

```console
PS C:\> scoop install xonotic
WARN  Purging previous failed installation of xonotic.
ERROR 'xonotic' isn't installed correctly.
Removing older version (0.8.5).
'xonotic' was uninstalled.
WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
WARN  To disable this warning, run 'scoop config aria2-warning-enabled false'.
Installing 'xonotic' (0.8.5) [64bit] from games bucket
Loading xonotic-0.8.5.zip from cache.
Checking hash of xonotic-0.8.5.zip ... ok.
Extracting xonotic-0.8.5.zip ... Could not find 'Xonotic'! (error 11)
At D:\scoop\apps\scoop\current\lib\core.ps1:638 char:9
+         throw "Could not find '$(fname $from)'! (error $($proc.ExitCo ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Could not find 'Xonotic'! (error 11):String) [], RuntimeException
    + FullyQualifiedErrorId : Could not find 'Xonotic'! (error 11)
```

### Possible Solution

no idea

### Scoop and Buckets Version

```console
Current Scoop version:
v0.3.1 - Released at 2022-11-15

'extras' bucket:
495b86ed1 (HEAD -> master, origin/master, origin/HEAD) sidekick-browser: Update to version 112.44.2.32807-769237f

'games' bucket:
aaac3287 (HEAD -> master, origin/master, origin/HEAD) yuzu-pineapple: Update to version EA-3631

'main' bucket:
808065f7d (HEAD -> master, origin/master, origin/HEAD) minio: Update to version 2023-06-02T23-17-26Z
```

### Scoop Config

```console
root_path    : D:\scoop
global_path  : D:\application\scoop
last_update  : 2023-06-03T16:40:05.6013966+05:30
scoop_branch : master
scoop_repo   : https://github.com/ScoopInstaller/Scoop
```

### PowerShell Version

```console
Name                           Value
----                           -----
PSVersion                      5.1.22621.963
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.22621.963
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by nonperforming** [comment]
This issue still exists.

Same steps to reproduce.

Current behavior
```pwsh
WARN  Purging previous failed installation of xonotic.
ERROR 'xonotic' isn't installed correctly.
Removing older version (0.8.5).
'xonotic' was uninstalled.
Installing 'xonotic' (0.8.5) [64bit] from games bucket
Loading xonotic-0.8.5.zip from cache.
Checking hash of xonotic-0.8.5.zip ... ok.
Extracting xonotic-0.8.5.zip ... Exception: C:\Users\itsaw\scoop\apps\scoop\current\lib\core.ps1:638
Line |
 638 |          throw "Could not find '$(fname $from)'! (error $($proc.ExitCo …
     |          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Could not find 'Xonotic'! (error 11)
```

Scoop and Buckets version
```
Current Scoop version:
v0.3.1 - Released at 2022-11-15

'extras' bucket:
1175f646c (HEAD -> master, origin/master, origin/HEAD) wsl2-distro-manager: Update to version 1.8.8

'foosel' bucket:
a3acbfe (HEAD -> main, origin/main, origin/HEAD) freecad-weekly: Update to version 33848

'games' bucket:
914a3ab7 (HEAD -> master, origin/master, origin/HEAD) yuzu-pineapple: Update to version EA-3824

'main' bucket:
0eb2f7685 (HEAD -> master, origin/master, origin/HEAD) intermodal: Update to version 0.1.13

'nirsoft' bucket:
a4b8597 (HEAD -> master, origin/master, origin/HEAD) Updated BrowserDownloadsView

'nonportable' bucket:
d229e0e (HEAD -> master, origin/master, origin/HEAD) 10x-np: Update to version 1.0.73

'versions' bucket:
9a9d4a5b0 (HEAD -> master, origin/master, origin/HEAD) neovim-nightly@0.10.0-dev-882-gd7ae9ae3e: Fix hash (Closes #1329)
```

Scoop Config
```
last_update            : 2023-08-20T20:23:24.1445061+12:00
aria2-warning-disabled : False
aria2-warning-enabled  : False
scoop_branch           : master
scoop_repo             : https://github.com/ScoopInstaller/Scoop
```

Powershell Version
```
Name                           Value
----                           -----
PSVersion                      5.1.22621.1778
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.22621.1778
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

Attempted with PowerShell built-into Windows 11, PowerShell 7 from scoop, and nushell from scoop

**Comment by Calinou** [comment]
Duplicate of https://github.com/Calinou/scoop-games/issues/877.
